test/test_retriever.py

```diff
@@ -149,10 +180,55 @@ def test_format_insert_value_string_simple():
 
 def test_format_insert_value_string_complex():
     """Test formatting of values for insert statements"""
-    assert test_engine.format_insert_value('my notes: "have extra, stuff"', 'char') == '\'my notes: \\"have extra, stuff\\"\''
+    assert test_engine.format_insert_value('my notes: "have extra, stuff"',
+                                           'char') == '\'my notes: \\"have extra, stuff\\"\''
 
 
 def test_getmd5():
```

Comment:
Since we have two of these functions let's specify the differences in the names: `test_getmd5_lines` and `test_getmd5_path`.